docs/docs/modules/indexes/vector_stores/integrations/elasticsearch.md

```diff
@@ -0,0 +1,132 @@
+---
```

Comment:
Best to make this a `.mdx` file and import code snippets from `examples/` but I'm happy to do this later